_security-plugin/configuration/multi-auth.md

```diff
@@ -0,0 +1,132 @@
+`opensearch_security.ui.basicauth.login.brandimage` |  Login button logo. Supported file types are SVG, PNG, and GIF.
+`opensearch_security.ui.basicauth.login.showbrandimage` |  Determines whether a logo for the login button is displayed or not. Default is `true`. 
+
+### OpenID Connect authentication settings
+
+These settings allow you to customize the sign-in button associated with OpenID Connect authentication. For the essential settings required to use OpenID Connect as a single sign-in option, see [OpenSearch Dashboards single sign-on]({{site.url}}{{site.baseurl}}/security-plugin/configuration/openid-connect/#opensearch-dashboards-single-sign-on).
+
+Setting | Description
+:--- | :--- |:--- |:--- |
+`opensearch_security.ui.openid.login.buttonname` |  Display name for the login button. "Log in with single sign-on" by default.
+`opensearch_security.ui.openid.login.brandimage` |  Login button logo. Supported file types are SVG, PNG, and GIF.
+`opensearch_security.ui.openid.login.showbrandimage` |  Determines whether a logo for the login button is displayed or not. Default is `false`.
+
+### SAML authentication settings
+
+These settings allow you to customize the sign-in button associated with SAML authentication. For the essential settings required to use SAML as a sign-in option, see [OpenSearch Dashboards configuration]({{site.url}}{{site.baseurl}}/security-plugin/configuration/saml/#opensearch-dashboards-configuration).
+
+Setting | Description
+:--- | :--- |:--- |:--- |
+`opensearch_security.ui.saml.login.buttonname` |  Display name for the login button. "Log in with single sign-on" by default.
+`opensearch_security.ui.saml.login.brandimage` |  Login button logo. Supported file types are SVG, PNG, and GIF.
+`opensearch_security.ui.saml.login.showbrandimage` |  Determines whether a logo for the login button is displayed or not. Default is `false`.
+
+## Sample setup
+The following example shows basic settings in the `opensearch_dashboards.yml` file  when configured for two types of authentication at sign-in.
+
+```yml
+# The several settings directly below are typical of all `opensearch_dashboards.yml` configurations. #
+server.host: 0.0.0.0
+server.port: 5601
+opensearch.hosts: ["https://localhost:9200"]
+opensearch.ssl.verificationMode: none
+opensearch.username: <preferred username>
+opensearch.password: <preferred password>
+opensearch.requestHeadersAllowlist: ["securitytenant","Authorization"]
+opensearch_security.multitenancy.enabled: true
+opensearch_security.multitenancy.tenants.preferred: ["Private", "Global"]
+opensearch_security.readonly_mode.roles: ["<role_for_read_only>"]
+
+# Settings that enable multiple option authentication in the sign-in window #
+opensearch_security.auth.multiple_auth_enabled: true
+opensearch_security.auth.type: ["basicauth","openid"]
+
+# Basic authentication customization #
+opensearch_security.ui.basicauth.login.buttonname: Log in
```

Comment:
Sorry for the confusion, basicauth authentication does not have the setting to customize the login buttonname - opensearch_security.ui.basicauth.login.buttonname. Please help remove it from the table.

Reply:
Removed this setting.